triggers/service/BUILD.bazel

```diff
@@ -111,6 +111,7 @@ binary_deps = [
     "//libs-scala/scala-utils",
     "//triggers/service/auth:middleware-api",
     "@maven//:org_slf4j_slf4j_api",
+    "//triggers/runner:trigger-runner-lib",
```

Comment:
These changes are necessary as this dependency does not appear to be transitively gained (thanks @akrmn for that).